### ISSUE TITLE
about training config

### ISSUE THREAD

**Issue by qingyshi** [UX-Decoder/Semantic-SAM#75]
What was the duration of your model's training, and could you provide details about your training configuration?

**Comment by xushilin1** [comment]
@FengLi-ust 
Hi, Thanks for your open-source code.
I am re-training your model using the SAM dataset with SWIN-L backbone. Could you please provide me with information about the duration of your training time, the specifications of the machine used, and the batch size?
Thanks a lot.

**Comment by FengLi-ust** [comment]
Yes, we train the whole SA-1B dataset for around 2 epochs. We use 64 V100 machines with batch size 64.

**Comment by JosephKJ** [comment]
Thank you for the response. How much time did it take?
